src/components/WebexMemberRoster/WebexMemberRoster.jsx

```diff
@@ -24,11 +24,37 @@ import useMemberships from '../hooks/useMemberships';
  */
 export default function WebexMemberRoster({destinationID, destinationType}) {
   const memberships = useMemberships(destinationID, destinationType);
-  const members = memberships.map(
+
+  const getMembersNode = (data) => (data || []).map(
```

Comment:
as this function returns a list of nodes, maybe rename it to `getMembers` or `renderMembers`, and also rename `data` to `members`.